docs-gen/content/docs/getting-started/installation.md

```diff
@@ -40,6 +40,10 @@ Valid configuration options to override the [Default `values.yaml`]({{< ghblob "
 | `manager.monitoring.rbacSecured`                          | Enable addtional RBAC-based security                                                                                                                                                                                |
 | `manager.monitoring.serviceMonitor.create`                | Create a ServiceMonitor for the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator)                                                                                                   |
 | `manager.monitoring.serviceMonitor.endpointConfiguration` | Additional configuration to be used in the service monitor endpoint (path, port and scheme are provided)                                                                                                            |
+| `manager.multiversion.enabled`                            | Enable multiversion API. Should be used in production to allow version migration. Disable for simplified installation                                                                                               |
+| `manager.multiversion.webhookCertificates.provider`       | K8s conversion webhook TLS certificate provider. Choose `cert-manager` for helm to deploy certificates if cert-manager is available. Choose `custom` otherwise                                                      |
+| `manager.multiversion.webhookCertificates.secretName`     | Name of a K8s secret deployed into the operator namespace to secure the webhook endpoint with. Required if the `custom` provider is chosen                                                                          |
+| `manager.multiversion.webhookCertificates.caBundle`       | CA bundle of the certificate authority that has signed the webhook's certificate. Required if the `custom` provider is chosen                                                                                       |
```

Comment:
```suggestion
| `manager.multiversion.enabled`                            | Enable multiversion API - should be enabled in production to allow version migration. Disable for simplified installation                                                                                               |
| `manager.multiversion.webhookCertificates.provider`       | K8s conversion webhook TLS certificate provider -choose `cert-manager` for Helm to deploy certificates if cert-manager is available or `custom` otherwise (see below)                                                      |
| `manager.multiversion.webhookCertificates.secretName`     | Name of a K8s secret deployed into the operator namespace to secure the webhook endpoint with, required if the `custom` provider is chosen                                                                          |
| `manager.multiversion.webhookCertificates.caBundle`       | CA bundle of the certificate authority that has signed the webhook's certificate, required if the `custom` provider is chosen                                                                                       |
```